interp/interp.mllib

```diff
@@ -13,7 +13,6 @@ Impargs
 Implicit_quantifiers
 Constrintern
 Modintern
-Constrextern
 Coqlib
```

Comment:
I think I will remove this commit. The original motivation was to remark that the only user of `constrextern` is the printing system, and in a many printing functions are done in externalization but the change is arbitrary and brings no benefit so it should go away.